src/pages/jobs.md

```diff
@@ -6,42 +6,19 @@ description: Shorebird openings
 
 # Open Positions
 
-Shorebird is not actively recruiting at this time.
+- [Developer Relations / Marketing Lead](#developer-relations--marketing-lead)
+- [Generalist Software Engineer](#generalist-software-engineer)
```

Comment:
I'd pull Generalist out of the title. I don't think thats what someone would put on their LinkedIn